CMakeLists.txt

```diff
@@ -125,19 +144,24 @@ endif()
 
 option(QUDA_DIRAC_WILSON "build Wilson Dirac operators" ${QUDA_DIRAC_DEFAULT})
 option(QUDA_DIRAC_CLOVER "build clover Dirac operators" ${QUDA_DIRAC_DEFAULT})
+# Dynamic inversion saves memory but decreases the flops
```

Comment:
This seems out of place? Also, increases flops, not decreases.

Reply:
Removed in d25026150